packages/map-template/src/components/Search/Search.jsx

```diff
@@ -144,8 +146,21 @@ function Search({ onLocationClick, categories, onLocationsFiltered, onSetSize, c
         }
     }, [currentVenueName]);
 
+    /*
+     * React on changes in categories prop.
+     */
+    useEffect(() => {
+        if (categories.length > 0) {
+            setHasCategories(true);
+        } else {
+            setHasCategories(false);
+        }
```

Comment:
Would this work as well?

```suggestion
        setHasCategories(categories.length > 0);
```

Reply:
Yes, that's nice simplification 👏